app/views/green_lanes/results/_exemptions_card.html.erb

```diff
@@ -8,7 +8,7 @@
   <div class="govuk-summary-card__content">
     <div class="govuk-summary-list">
       <% @assessments.public_send("cat#{category}_with_exemptions").each do |category_assessment| %>
-        <% category_assessment.exemptions.each do |exemption| %>
+        <% category_assessment.exemptions.select { |exemption| dig_category_answer(@answers, category, category_assessment.category_assessment_id).include?(exemption.code) }.each do |exemption| %>
```

Comment:
This is just filtering to only show the exemptions that have been met when you have multiple for a category assessment

Reply:
Yeah i agree, to be honest this part of the code i need to double check with Chris and will see what he says after the UR sessions. its something i dont think is portrayed very easily on the tickets.